packages/ComponentLibrary/src/components/Modal/modal.constants.ts

```diff
@@ -0,0 +1,8 @@
+export const MENU_ITEMS = [
```

Comment:
Item names should be in English. Take JSON type translations to a file as constants